recipes/tsl-hat-trie/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.28.0"
+
+class TslHatTrieConan(ConanFile):
+    name = "tsl-hat-trie"
+    license = "MIT"
+    description = "C++ implementation of a fast and memory efficient HAT-trie."
+    topics = ("conan", "string", "trie", "structure", "hash map", "hash set")
+    homepage = "https://github.com/Tessil/hat-trie"
+    url = "https://github.com/conan-io/conan-center-index"
+    requires = "tsl-array-hash/0.7.1"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("hat-trie-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        tools.rmdir(os.path.join(self._source_subfolder, "include", "tsl", "array-hash"))
+        tools.replace_in_file(os.path.join(self._source_subfolder, "include", "tsl", "htrie_hash.h"), "array-hash/", "tsl/")
```

Comment:
```suggestion
        tools.replace_in_file(os.path.join(self._source_subfolder, "include", "tsl", "htrie_hash.h"), '#include "array-hash/', '#include "tsl/')
```

Let's be a bit more strict (this won't break in the future with higher probability).